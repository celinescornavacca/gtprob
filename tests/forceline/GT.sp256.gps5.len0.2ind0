(((((((DA1,DA2),((DA3,DA4),DA0)),(((((((IF2,(IF3,IF4)),(IF0,IF1)),(((JF3,JF4),JF0),(JF1,JF2))),((AC2,(AC3,AC4)),(AC0,AC1))),((((((RD3,RD4),(RD0,(RD1,RD2))),(SD0,(SD1,(SD2,(SD3,SD4))))),((ID1,((ID3,ID4),ID2)),ID0)),((OC0,(OC1,((OC3,OC4),OC2))),(PC0,((PC2,(PC3,PC4)),PC1)))),(((((PI0,((PI3,PI4),(PI1,PI2))),(QI0,((QI1,QI2),(QI3,QI4)))),((EF0,(EF1,EF2)),(EF3,EF4))),(((QC3,QC4),(QC1,QC2)),QC0)),((((KB3,KB4),KB0),(KB1,KB2)),(((LB3,LB4),LB0),(LB1,LB2)))))),((IA0,IA1),((IA3,IA4),IA2))),(((TD3,TD4),((TD1,TD2),TD0)),((((UD3,UD4),UD2),UD1),UD0)))),(((((((SA1,((SA3,SA4),SA2)),SA0),((((AG0,(AG3,AG4)),(AG1,AG2)),(BG0,((BG3,BG4),(BG1,BG2)))),((CC1,CC2),((CC3,CC4),CC0)))),(((((((((ZA1,ZA2),(ZA3,ZA4)),ZA0),((AB3,AB4),(AB0,(AB1,AB2)))),((((((MD3,MD4),MD2),(MD0,MD1)),(((ZE1,ZE2),(ZE0,(ZE3,ZE4))),(((AF3,AF4),AF2),(AF0,AF1)))),((QJ0,(QJ1,(QJ2,(QJ3,QJ4)))),(((RJ3,RJ4),RJ0),(RJ1,RJ2)))),(((JD1,JD2),(JD0,(JD3,JD4))),((KD0,KD1),(KD2,(KD3,KD4)))))),(((LG1,LG2),(LG0,(LG3,LG4))),(((MG1,MG2),MG0),(MG3,MG4)))),(((((VB3,VB4),VB0),(VB1,VB2)),(WB0,(((WB3,WB4),WB2),WB1))),((QA3,QA4),((QA1,QA2),QA0)))),(((((HB1,HB2),HB0),(HB3,HB4)),((((ZD1,ZD2),(ZD3,ZD4)),ZD0),((AE0,(AE1,AE2)),(AE3,AE4)))),((KI0,(KI1,((KI3,KI4),KI2))),((LI1,((LI3,LI4),LI2)),LI0)))),(((NA0,(NA1,NA2)),(NA3,NA4)),(((((PD0,PD1),(PD2,(PD3,PD4))),((QD0,(QD1,QD2)),(QD3,QD4))),(((((VE3,VE4),(VE1,VE2)),VE0),(WE0,(WE1,((WE3,WE4),WE2)))),((CD1,((CD3,CD4),CD2)),CD0))),((((UA3,UA4),UA2),UA1),UA0))))),((((((BE3,BE4),BE2),BE1),BE0),(CE0,(((CE3,CE4),CE2),CE1))),((((EH1,EH2),(EH0,(EH3,EH4))),(((CI0,CI1),(CI2,(CI3,CI4))),(((DI3,DI4),(DI1,DI2)),DI0))),(TF0,((TF2,(TF3,TF4)),TF1))))),(((XB1,XB2),((XB3,XB4),XB0)),(((YB1,YB2),YB0),(YB3,YB4)))),((((((WG0,(WG1,WG2)),(WG3,WG4)),((((CH1,CH2),(CH3,CH4)),CH0),((DH1,DH2),((DH3,DH4),DH0)))),(((((((SC3,SC4),SC2),SC1),SC0),(((TC1,TC2),(TC3,TC4)),TC0)),(((GC2,(GC3,GC4)),GC1),GC0)),((UF0,(((UF3,UF4),UF2),UF1)),(((VF1,VF2),(VF3,VF4)),VF0)))),(((((LF1,LF2),(LF3,LF4)),LF0),(((((AH3,AH4),(AH1,AH2)),AH0),((BH2,(BH3,BH4)),(BH0,BH1))),(((WF1,WF2),(WF3,WF4)),WF0))),((((((ZB2,(ZB3,ZB4)),ZB1),ZB0),((RE0,((RE1,RE2),(RE3,RE4))),(SE0,((SE3,SE4),(SE1,SE2))))),((((UB3,UB4),UB2),(UB0,UB1)),(((LE0,LE1),((LE3,LE4),LE2)),((ME0,ME1),(ME2,(ME3,ME4)))))),((TA0,TA1),(TA2,(TA3,TA4)))))),((((XG0,((XG3,XG4),(XG1,XG2))),(((YG1,YG2),(YG3,YG4)),YG0)),((PE0,((PE3,PE4),(PE1,PE2))),((QE0,(QE3,QE4)),(QE1,QE2)))),((((((RH0,(RH1,RH2)),(RH3,RH4)),(((SH3,SH4),(SH1,SH2)),SH0)),((((OD3,OD4),OD2),OD1),OD0)),((((II0,(((II3,II4),II2),II1)),(((JI1,JI2),JI0),(JI3,JI4))),((((((CJ1,CJ2),CJ0),(CJ3,CJ4)),((DJ1,DJ2),(DJ0,(DJ3,DJ4)))),((ZI0,(ZI1,ZI2)),(ZI3,ZI4))),((TH1,TH2),((TH3,TH4),TH0)))),((VI0,((VI3,VI4),(VI1,VI2))),((WI0,(WI3,WI4)),(WI1,WI2))))),(((((IH3,IH4),IH0),(IH1,IH2)),((JH0,JH1),(JH2,(JH3,JH4)))),((IJ0,(IJ1,(IJ2,(IJ3,IJ4)))),(JJ0,((JJ1,JJ2),(JJ3,JJ4)))))))))),(((BA0,(BA1,BA2)),(BA3,BA4)),(((((UJ0,((UJ2,(UJ3,UJ4)),UJ1)),(((VJ1,VJ2),(VJ3,VJ4)),VJ0)),((((FB3,FB4),(FB1,FB2)),FB0),((((((EI3,EI4),EI0),(EI1,EI2)),((FI0,FI1),(FI2,(FI3,FI4)))),(VH0,(VH1,((VH3,VH4),VH2)))),((GB2,(GB3,GB4)),(GB0,GB1))))),(((BD0,(BD3,BD4)),(BD1,BD2)),((EJ0,(EJ1,(EJ2,(EJ3,EJ4)))),((FJ2,(FJ3,FJ4)),(FJ0,FJ1))))),((((((MI3,MI4),(MI0,(MI1,MI2))),(NI0,(((NI3,NI4),NI2),NI1))),(((GF1,GF2),GF0),(GF3,GF4))),(((HA3,HA4),HA2),(HA0,HA1))),(((((MF3,MF4),(MF0,(MF1,MF2))),(((NF3,NF4),NF2),(NF0,NF1))),(((FC1,FC2),(FC0,(FC3,FC4))),(((RI0,(RI3,RI4)),(RI1,RI2)),((SI0,SI1),((SI3,SI4),SI2))))),(((((DE3,DE4),(DE1,DE2)),DE0),(((EE3,EE4),EE2),(EE0,EE1))),((XD0,(XD1,XD2)),(XD3,XD4)))))))),((((((((XC3,XC4),XC0),(XC1,XC2)),(YC0,(((YC3,YC4),YC2),YC1))),((((((HC3,HC4),HC2),HC1),HC0),(((VD1,VD2),((VD3,VD4),VD0)),((WD1,(WD2,(WD3,WD4))),WD0))),((((PH0,(PH1,((PH3,PH4),PH2))),((QH0,QH1),((QH3,QH4),QH2))),((TB3,TB4),(TB0,(TB1,TB2)))),(((EB2,(EB3,EB4)),(EB0,EB1)),(((((PG1,PG2),(PG3,PG4)),PG0),((QG0,QG1),(QG2,(QG3,QG4)))),((FF0,FF1),((FF3,FF4),FF2))))))),((((((((IC1,IC2),IC0),(IC3,IC4)),((UC0,(((UC3,UC4),UC2),UC1)),(VC0,((VC2,(VC3,VC4)),VC1)))),((BC2,(BC3,BC4)),(BC0,BC1))),((((OF3,OF4),(OF1,OF2)),OF0),(PF0,((PF1,PF2),(PF3,PF4))))),(((DC3,DC4),(DC0,(DC1,DC2))),((EC3,EC4),(EC0,(EC1,EC2))))),(((EA1,EA2),EA0),(EA3,EA4)))),((((((FE0,(FE3,FE4)),(FE1,FE2)),((GE0,(GE1,GE2)),(GE3,GE4))),(((PA3,PA4),(PA1,PA2)),PA0)),((((QF1,QF2),((QF3,QF4),QF0)),(((HG3,HG4),(HG0,(HG1,HG2))),(IG0,((IG3,IG4),(IG1,IG2))))),((OA3,OA4),((OA1,OA2),OA0)))),(((((RA3,RA4),RA2),RA1),RA0),((((BF0,((BF2,(BF3,BF4)),BF1)),((XF0,((XF3,XF4),(XF1,XF2))),((SG0,(SG1,((SG3,SG4),SG2))),((TG0,TG1),((TG3,TG4),TG2))))),((((DD3,DD4),DD2),(DD0,DD1)),(ED0,(ED1,(ED2,(ED3,ED4)))))),(((((SJ1,(SJ2,(SJ3,SJ4))),SJ0),(((TJ1,TJ2),TJ0),(TJ3,TJ4))),((((AI2,(AI3,AI4)),AI1),AI0),(((BI3,BI4),BI0),(BI1,BI2)))),(((FG1,FG2),(FG0,(FG3,FG4))),((GG0,GG1),(GG2,(GG3,GG4))))))))),(((((((((CG1,(CG2,(CG3,CG4))),CG0),((DG1,DG2),((DG3,DG4),DG0))),((((((HE2,(HE3,HE4)),HE1),HE0),((IE0,IE1),((IE3,IE4),IE2))),(((JC3,JC4),(JC1,JC2)),JC0)),(((((KC3,KC4),KC2),KC1),KC0),(((((RG3,RG4),(RG1,RG2)),RG0),(((MH2,(MH3,MH4)),(MH0,MH1)),((AJ0,((AJ3,AJ4),(AJ1,AJ2))),((BJ0,BJ1),((BJ3,BJ4),BJ2))))),((((((YH3,YH4),YH2),YH1),YH0),(ZH0,((ZH1,ZH2),(ZH3,ZH4)))),((HF3,HF4),(HF0,(HF1,HF2)))))))),(((YA0,YA1),(YA2,(YA3,YA4))),((ZG0,((ZG2,(ZG3,ZG4)),ZG1)),(((WH1,WH2),(WH0,(WH3,WH4))),(XH0,((XH1,XH2),(XH3,XH4))))))),(((((((MJ3,MJ4),MJ0),(MJ1,MJ2)),((NJ2,(NJ3,NJ4)),(NJ0,NJ1))),(OI0,((OI2,(OI3,OI4)),OI1))),(JE0,((JE3,JE4),(JE1,JE2)))),(YD0,(((YD3,YD4),YD2),YD1)))),(((((((EG3,EG4),EG2),EG1),EG0),(((NH0,(NH1,NH2)),(NH3,NH4)),((OH0,OH1),(OH2,(OH3,OH4))))),((((GI1,GI2),GI0),(GI3,GI4)),(((HI2,(HI3,HI4)),HI1),HI0))),((((BB2,(BB3,BB4)),(BB0,BB1)),(CB0,(CB1,(CB2,(CB3,CB4))))),(((OJ1,OJ2),(OJ0,(OJ3,OJ4))),(PJ0,(PJ1,((PJ3,PJ4),PJ2))))))),(((((((MC3,MC4),MC2),MC1),MC0),(NC0,(NC1,(NC2,(NC3,NC4))))),((KA0,(KA1,((KA3,KA4),KA2))),((LA3,LA4),((LA1,LA2),LA0)))),(((((((TE1,(TE2,(TE3,TE4))),TE0),(((XI0,(XI1,XI2)),(XI3,XI4)),(((YI1,YI2),(YI3,YI4)),YI0))),((((LC0,(LC1,LC2)),(LC3,LC4)),(((UG0,UG1),(UG2,(UG3,UG4))),((VG1,((VG3,VG4),VG2)),VG0))),((MB0,(MB3,MB4)),(MB1,MB2)))),(((JA1,JA2),JA0),(JA3,JA4))),((((((FH3,FH4),FH2),(FH0,FH1)),(((GH1,GH2),GH0),(GH3,GH4))),(((ND1,(ND2,(ND3,ND4))),ND0),(((((KJ3,KJ4),KJ2),(KJ0,KJ1)),(LJ0,(((LJ3,LJ4),LJ2),LJ1))),((UH0,UH1),((UH3,UH4),UH2))))),(XA0,(XA1,(XA2,(XA3,XA4)))))),(((((((IB3,IB4),(IB1,IB2)),IB0),((JB1,(JB2,(JB3,JB4))),JB0)),((((KH1,KH2),KH0),(KH3,KH4)),(((LH2,(LH3,LH4)),LH1),LH0))),(((RB3,RB4),((RB1,RB2),RB0)),(SB0,((SB1,SB2),(SB3,SB4))))),(((JG0,((JG1,JG2),(JG3,JG4))),(((KG2,(KG3,KG4)),KG1),KG0)),(((DB0,DB1),((DB3,DB4),DB2)),(((CF0,CF1),((CF3,CF4),CF2)),(((DF2,(DF3,DF4)),DF1),DF0)))))))),(((AA0,(AA1,(AA2,(AA3,AA4)))),(((GA0,GA1),(GA2,(GA3,GA4))),((((TI1,(TI2,(TI3,TI4))),TI0),((UI1,((UI3,UI4),UI2)),UI0)),(((MA3,MA4),MA0),(MA1,MA2))))),((FA0,((FA2,(FA3,FA4)),FA1)),((((PB3,PB4),(PB1,PB2)),PB0),(((ZC3,ZC4),(ZC0,(ZC1,ZC2))),((AD1,AD2),((AD3,AD4),AD0))))))))),((((((((NE1,NE2),(NE0,(NE3,NE4))),(OE0,(OE1,(OE2,(OE3,OE4))))),(((QB0,QB1),((QB3,QB4),QB2)),((((((XE0,((XE1,XE2),(XE3,XE4))),(((YE1,YE2),(YE3,YE4)),YE0)),((UE1,UE2),(UE0,(UE3,UE4)))),(KE0,((KE1,KE2),(KE3,KE4)))),((LD0,LD1),((LD3,LD4),LD2))),((RC2,(RC3,RC4)),(RC0,RC1))))),((((WC3,WC4),WC0),(WC1,WC2)),(((NG3,NG4),(NG0,(NG1,NG2))),(OG0,((OG2,(OG3,OG4)),OG1))))),(((((FD1,FD2),FD0),(FD3,FD4)),(((GD1,((GD3,GD4),GD2)),GD0),((HD1,HD2),((HD3,HD4),HD0)))),((((((((YF3,YF4),YF2),YF1),YF0),((ZF0,(ZF1,ZF2)),(ZF3,ZF4))),(((RF0,((RF1,RF2),(RF3,RF4))),(((SF3,SF4),(SF1,SF2)),SF0)),(KF0,((KF2,(KF3,KF4)),KF1)))),(WA0,((WA1,WA2),(WA3,WA4)))),(VA0,((VA1,VA2),(VA3,VA4)))))),(((NB0,(((NB3,NB4),NB2),NB1)),((OB0,OB1),((OB3,OB4),OB2))),((HH0,(HH1,((HH3,HH4),HH2))),(((GJ1,((GJ3,GJ4),GJ2)),GJ0),(HJ0,((HJ2,(HJ3,HJ4)),HJ1)))))),((CA1,((CA3,CA4),CA2)),CA0)));
