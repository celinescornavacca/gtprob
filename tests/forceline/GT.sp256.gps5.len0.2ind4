((((((((DF0,((DF1,DF2),(DF3,DF4))),((EF0,(EF3,EF4)),(EF1,EF2))),((OG0,(OG1,(OG2,(OG3,OG4)))),((((PH1,PH2),(PH3,PH4)),PH0),(QH0,(((QH3,QH4),QH2),QH1))))),(((ZA3,ZA4),ZA2),(ZA0,ZA1))),(((BA0,(BA3,BA4)),(BA1,BA2)),(((RD1,RD2),((RD3,RD4),RD0)),(((HF1,((HF3,HF4),HF2)),HF0),((IF0,IF1),(IF2,(IF3,IF4))))))),(((DA3,DA4),(DA0,(DA1,DA2))),((((HA3,HA4),(HA0,(HA1,HA2))),((((((VF3,VF4),VF2),VF1),VF0),(((WF3,WF4),WF2),(WF0,WF1))),((PB1,(PB2,(PB3,PB4))),PB0))),(((XB1,XB2),(XB0,(XB3,XB4))),(((YB0,YB1),((YB3,YB4),YB2)),((ZB3,ZB4),(ZB0,(ZB1,ZB2)))))))),((((((((JE0,((JE3,JE4),(JE1,JE2))),(KE0,(KE1,((KE3,KE4),KE2)))),(((PG2,(PG3,PG4)),(PG0,PG1)),(QG0,(QG1,(QG2,(QG3,QG4)))))),((((XE0,((XE1,XE2),(XE3,XE4))),((((YE3,YE4),YE2),YE1),YE0)),(WC0,((WC1,WC2),(WC3,WC4)))),((IA0,((IA2,(IA3,IA4)),IA1)),((MB0,(MB1,(MB2,(MB3,MB4)))),((IE0,(IE1,((IE3,IE4),IE2))),(((TF3,TF4),((TF1,TF2),TF0)),((UF0,UF1),((UF3,UF4),UF2)))))))),(((((EC0,EC1),((EC3,EC4),EC2)),(((CG0,CG1),((CG3,CG4),CG2)),((DG1,((DG3,DG4),DG2)),DG0))),(((((((((KB3,KB4),KB2),KB1),KB0),(LB0,(((LB3,LB4),LB2),LB1))),((((JC3,JC4),JC0),(JC1,JC2)),((((OJ0,(OJ3,OJ4)),(OJ1,OJ2)),(PJ0,(((PJ3,PJ4),PJ2),PJ1))),(((LI3,LI4),(LI1,LI2)),LI0)))),((MA0,MA1),(MA2,(MA3,MA4)))),(((LA3,LA4),LA0),(LA1,LA2))),(((OD0,(OD1,(OD2,(OD3,OD4)))),(((BE2,(BE3,BE4)),(BE0,BE1)),(CE0,(((CE3,CE4),CE2),CE1)))),((((DJ0,DJ1),(DJ2,(DJ3,DJ4))),((EJ1,((EJ3,EJ4),EJ2)),EJ0)),(((NG1,NG2),NG0),(NG3,NG4)))))),(((((KC3,KC4),((KC1,KC2),KC0)),((((QF3,QF4),(QF0,(QF1,QF2))),(((RF1,RF2),RF0),(RF3,RF4))),(((FE3,FE4),FE0),(FE1,FE2)))),(BB0,(BB1,(BB2,(BB3,BB4))))),(((((SD2,(SD3,SD4)),SD1),SD0),((((GH0,(GH3,GH4)),(GH1,GH2)),(((HH1,HH2),HH0),(HH3,HH4))),((((XG2,(XG3,XG4)),XG1),XG0),((YG0,YG1),(YG2,(YG3,YG4)))))),((QE0,((QE3,QE4),(QE1,QE2))),(((((LG3,LG4),LG2),LG1),LG0),((MG1,MG2),(MG0,(MG3,MG4))))))))),((((AB0,((AB1,AB2),(AB3,AB4))),((((NH2,(NH3,NH4)),(NH0,NH1)),((OH3,OH4),(OH0,(OH1,OH2)))),((((XI0,XI1),((XI3,XI4),XI2)),((YI0,(YI3,YI4)),(YI1,YI2))),(ZG0,(((ZG3,ZG4),ZG2),ZG1))))),((((ZD2,(ZD3,ZD4)),ZD1),ZD0),(((AE1,AE2),AE0),(AE3,AE4)))),(((((DD0,(((DD3,DD4),DD2),DD1)),(((ED3,ED4),(ED1,ED2)),ED0)),((QB0,((QB1,QB2),(QB3,QB4))),((((BD0,BD1),((BD3,BD4),BD2)),(CD0,(CD1,((CD3,CD4),CD2)))),(TB0,((TB3,TB4),(TB1,TB2)))))),(((((KF3,KF4),KF0),(KF1,KF2)),((LF0,LF1),((LF3,LF4),LF2))),((HD1,HD2),(HD0,(HD3,HD4))))),((((((EG0,(EG1,EG2)),(EG3,EG4)),(FG0,(((FG3,FG4),FG2),FG1))),(((GD1,GD2),GD0),(GD3,GD4))),((DB1,(DB2,(DB3,DB4))),DB0)),(((((((IC1,IC2),IC0),(IC3,IC4)),((((QC1,QC2),QC0),(QC3,QC4)),((((OF3,OF4),(OF1,OF2)),OF0),(((PF3,PF4),PF0),(PF1,PF2))))),(((MF1,MF2),((MF3,MF4),MF0)),((NF2,(NF3,NF4)),(NF0,NF1)))),((((((ZC3,ZC4),ZC0),(ZC1,ZC2)),(AD0,(AD1,((AD3,AD4),AD2)))),(((WB3,WB4),WB0),(WB1,WB2))),((TA3,TA4),((TA1,TA2),TA0)))),(((KA1,KA2),(KA3,KA4)),KA0)))))),((((((AC1,(AC2,(AC3,AC4))),AC0),((BC0,BC1),((BC3,BC4),BC2))),((XC0,(XC1,(XC2,(XC3,XC4)))),(((YC3,YC4),(YC1,YC2)),YC0))),((QA1,(QA2,(QA3,QA4))),QA0)),(((((((BG3,BG4),BG2),BG1),BG0),(((AH1,(AH2,(AH3,AH4))),AH0),(((TI0,TI1),((TI3,TI4),TI2)),(UI0,((UI1,UI2),(UI3,UI4)))))),(((MJ0,(((MJ3,MJ4),MJ2),MJ1)),((NJ3,NJ4),((NJ1,NJ2),NJ0))),((((FJ3,FJ4),FJ2),FJ1),FJ0))),((RE0,(RE1,(RE2,(RE3,RE4)))),((XH0,(XH1,((XH3,XH4),XH2))),((YH1,((YH3,YH4),YH2)),YH0)))))),(((((((BF0,((BF2,(BF3,BF4)),BF1)),((CF1,CF2),(CF0,(CF3,CF4)))),(((EB1,EB2),((EB3,EB4),EB0)),(((FB3,FB4),FB0),(FB1,FB2)))),(((RG0,RG1),(RG2,(RG3,RG4))),((SG2,(SG3,SG4)),(SG0,SG1)))),((((CB0,(CB1,CB2)),(CB3,CB4)),((VD0,((VD1,VD2),(VD3,VD4))),(((WD1,WD2),(WD3,WD4)),WD0))),((EA0,(EA1,(EA2,(EA3,EA4)))),(((VB0,VB1),(VB2,(VB3,VB4))),((((UJ2,(UJ3,UJ4)),UJ1),UJ0),(((VJ1,VJ2),(VJ3,VJ4)),VJ0)))))),(((((((GA3,GA4),GA2),GA1),GA0),((JA0,((JA1,JA2),(JA3,JA4))),((((ZH0,((ZH1,ZH2),(ZH3,ZH4))),((((AI3,AI4),AI2),AI1),AI0)),(((JB2,(JB3,JB4)),JB1),JB0)),(((((((FH3,FH4),FH2),(FH0,FH1)),(((VH1,VH2),(VH0,(VH3,VH4))),(((WH3,WH4),WH2),(WH0,WH1)))),(WE0,(WE1,(WE2,(WE3,WE4))))),(((PD1,PD2),(PD3,PD4)),PD0)),((UB0,UB1),((UB3,UB4),UB2)))))),(((((FF1,FF2),(FF3,FF4)),FF0),(((GF2,(GF3,GF4)),GF1),GF0)),(((((GE3,GE4),GE2),GE1),GE0),(HE0,((HE1,HE2),(HE3,HE4)))))),(((MC0,(MC1,(MC2,(MC3,MC4)))),((((IH3,IH4),(IH0,(IH1,IH2))),(((JH2,(JH3,JH4)),JH1),JH0)),(((((MI2,(MI3,MI4)),MI1),MI0),((NI0,(NI3,NI4)),(NI1,NI2))),((((CI3,CI4),CI2),CI1),CI0)))),((PA0,(((PA3,PA4),PA2),PA1)),((((BH0,((BH1,BH2),(BH3,BH4))),((CH1,CH2),((CH3,CH4),CH0))),((XD3,XD4),(XD0,(XD1,XD2)))),((RC0,(RC3,RC4)),(RC1,RC2))))))),(((((OC3,OC4),(OC1,OC2)),OC0),((PC1,(PC2,(PC3,PC4))),PC0)),(((SE0,(SE1,((SE3,SE4),SE2))),((TE1,((TE3,TE4),TE2)),TE0)),(LD0,(LD1,((LD3,LD4),LD2)))))))),(((((((AA3,AA4),((AA1,AA2),AA0)),(((UH1,UH2),(UH0,(UH3,UH4))),((VI0,(VI1,((VI3,VI4),VI2))),(WI0,(WI1,(WI2,(WI3,WI4))))))),((((((LE1,LE2),((LE3,LE4),LE0)),(((ME3,ME4),ME0),(ME1,ME2))),((((TD1,TD2),((TD3,TD4),TD0)),(((UD3,UD4),UD0),(UD1,UD2))),(QD0,((QD3,QD4),(QD1,QD2))))),((FD1,FD2),(FD0,(FD3,FD4)))),(((IB1,((IB3,IB4),IB2)),IB0),((MH0,(((MH3,MH4),MH2),MH1)),((((DI3,DI4),(DI1,DI2)),DI0),((EI3,EI4),(EI0,(EI1,EI2)))))))),(((((YD0,(((YD3,YD4),YD2),YD1)),(((BJ1,BJ2),(BJ0,(BJ3,BJ4))),(CJ0,((CJ3,CJ4),(CJ1,CJ2))))),(((HB0,HB1),((HB3,HB4),HB2)),(((((RI1,(RI2,(RI3,RI4))),RI0),((SI1,SI2),((SI3,SI4),SI0))),(((DH1,((DH3,DH4),DH2)),DH0),(((EH3,EH4),EH2),(EH0,EH1)))),((((LC3,LC4),LC2),LC1),LC0)))),(((NE0,(NE1,((NE3,NE4),NE2))),(OE0,(OE1,(OE2,(OE3,OE4))))),(YA0,((YA1,YA2),(YA3,YA4))))),((((((JI3,JI4),(JI0,(JI1,JI2))),(KI0,((KI3,KI4),(KI1,KI2)))),(((RA1,RA2),(RA3,RA4)),RA0)),(((((((RB1,RB2),(RB3,RB4)),RB0),(((JD0,((JD1,JD2),(JD3,JD4))),((KD1,KD2),(KD0,(KD3,KD4)))),(VC0,(VC1,(VC2,(VC3,VC4)))))),((((NB3,NB4),(NB1,NB2)),NB0),(((OB3,OB4),OB0),(OB1,OB2)))),((((((UE1,(UE2,(UE3,UE4))),UE0),(((VE1,VE2),(VE3,VE4)),VE0)),(((((JF1,JF2),(JF3,JF4)),JF0),(((((TG2,(TG3,TG4)),TG1),TG0),(UG0,((UG1,UG2),(UG3,UG4)))),(((KH1,KH2),((KH3,KH4),KH0)),((LH1,LH2),((LH3,LH4),LH0))))),(((ZF0,((ZF1,ZF2),(ZF3,ZF4))),(AG0,(AG1,((AG3,AG4),AG2)))),((SF3,SF4),((SF1,SF2),SF0))))),((GB1,GB2),(GB0,(GB3,GB4)))),((OA1,((OA3,OA4),OA2)),OA0))),((((((NC3,NC4),NC2),NC1),NC0),((((ZI3,ZI4),ZI0),(ZI1,ZI2)),(AJ0,(((AJ3,AJ4),AJ2),AJ1)))),(((UA1,UA2),(UA3,UA4)),UA0)))),((((CA3,CA4),CA2),(CA0,CA1)),((((GG0,((GG2,(GG3,GG4)),GG1)),((HG1,((HG3,HG4),HG2)),HG0)),((AF0,(AF3,AF4)),(AF1,AF2))),(((DE1,((DE3,DE4),DE2)),DE0),((EE2,(EE3,EE4)),(EE0,EE1)))))))),((((((((HC0,(HC3,HC4)),(HC1,HC2)),(((MD3,MD4),((MD1,MD2),MD0)),(((ND1,ND2),(ND3,ND4)),ND0))),(((((XF3,XF4),XF2),(XF0,XF1)),(((YF2,(YF3,YF4)),YF1),YF0)),(((FC1,FC2),((FC3,FC4),FC0)),(((GC2,(GC3,GC4)),GC1),GC0)))),((((((TC2,(TC3,TC4)),TC1),TC0),((UC3,UC4),(UC0,(UC1,UC2)))),((SC0,(SC3,SC4)),(SC1,SC2))),(((((QJ3,QJ4),QJ2),(QJ0,QJ1)),(RJ0,(RJ1,((RJ3,RJ4),RJ2)))),((OI3,OI4),(OI0,(OI1,OI2)))))),(FA0,((FA1,FA2),(FA3,FA4)))),(((VA1,VA2),((VA3,VA4),VA0)),((((WA1,WA2),WA0),(WA3,WA4)),(((((VG0,(VG3,VG4)),(VG1,VG2)),(WG0,(WG1,(WG2,(WG3,WG4))))),((PE1,PE2),((PE3,PE4),PE0))),((KG0,((KG2,(KG3,KG4)),KG1)),(((PI1,(PI2,(PI3,PI4))),PI0),((QI1,(QI2,(QI3,QI4))),QI0))))))),((((((SA2,(SA3,SA4)),(SA0,SA1)),(((XA0,XA1),(XA2,(XA3,XA4))),((FI0,((FI3,FI4),(FI1,FI2))),(GI0,(((GI3,GI4),GI2),GI1))))),((((ZE0,(((ZE3,ZE4),ZE2),ZE1)),((RH0,(((RH3,RH4),RH2),RH1)),((SH2,(SH3,SH4)),(SH0,SH1)))),(((ID3,ID4),ID2),(ID0,ID1))),(((NA3,NA4),(NA1,NA2)),NA0))),((((((GJ3,GJ4),GJ0),(GJ1,GJ2)),((HJ0,HJ1),((HJ3,HJ4),HJ2))),((((DC3,DC4),DC2),DC1),DC0)),((((SJ0,((SJ3,SJ4),(SJ1,SJ2))),((TJ1,TJ2),((TJ3,TJ4),TJ0))),((BI0,BI1),(BI2,(BI3,BI4)))),((SB3,SB4),(SB0,(SB1,SB2)))))),((((IG0,(IG3,IG4)),(IG1,IG2)),(JG0,((JG3,JG4),(JG1,JG2)))),((((IJ0,IJ1),((IJ3,IJ4),IJ2)),((JJ1,(JJ2,(JJ3,JJ4))),JJ0)),(((((HI3,HI4),HI2),HI1),HI0),(II0,(((II3,II4),II2),II1)))))))),(((TH0,(TH1,((TH3,TH4),TH2))),((((KJ3,KJ4),KJ2),(KJ0,KJ1)),((LJ3,LJ4),((LJ1,LJ2),LJ0)))),((CC3,CC4),((CC1,CC2),CC0)))));
