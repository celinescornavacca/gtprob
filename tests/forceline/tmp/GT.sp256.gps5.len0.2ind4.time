15475.28user 5.64system 4:18:07elapsed 99%CPU (0avgtext+0avgdata 2498848maxresident)k
40inputs+8outputs (0major+1635193minor)pagefaults 0swaps
