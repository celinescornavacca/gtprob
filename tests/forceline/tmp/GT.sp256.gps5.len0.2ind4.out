-2503.68537036
